### PR TITLE
updated env id as per ak env

### PR DESCRIPTION
**Pull request by vijay8451**
**- Updated env id as per AK_env:**
**- Below as per log:**
+++++++++++++++++++++
Organization","label":"Default_Organization","id":1},"created_at":"2019-01-11 16:20:52 UTC","updated_at":"2019-01-11 16:20:52 UTC","environments":[{"id":1,"name":"Library","label":"Library","permissions":{"readable":true}},{"id":3,"name":"Dev","label":"Dev","permissions":{"readable":true}}],"repositories":
+++++++++++++++++++++

### CODE REVIEW

**Comment by vijay8451** [comment]
Fixed #6639

**Comment by ntkathole** [comment]
Fixes #6639

File: tests/upgrades/test_capsule.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def test_pre_user_scenario_capsule_sync(self):
         content_view = content_view.update(['repository'])
         content_view.publish()
         promote(content_view.read().version[0], ak_env.id)
-        self.assertEqual(content_view.read().environment[0].id, ak_env.id)
+        self.assertEqual(content_view.read().environment[1].id, ak_env.id)
```
Comment (jyejare) [inline tests/upgrades/test_capsule.py]: Not, 1 but it should be -1.

File: tests/upgrades/test_capsule.py [hunk 2]
```diff
@@ -214,7 +214,7 @@ def test_post_user_scenario_capsule_sync_2(self):
...
```
Comment (jyejare) [inline tests/upgrades/test_capsule.py]: same here

**Comment by vijay8451** [comment]
Thanks @jyejare it's there now

**Pull request merged**